devUtils.js

```diff
@@ -0,0 +1,23 @@
+const isProd = process.env.NODE_ENV === "production";
+
+/**
+ * Logs messages to console only in development environments
+ * @param {...any} args - Arguments to pass to console.log
+ */
+const devLog = (...args) => {
```

Comment:
I like this implementation better than my suggestion of `devLog(type, ...args)` :)